client/blocks/reader-post-card/featured-asset.jsx

```diff
@@ -2,37 +2,65 @@
  * External Dependencies
  */
 import React from 'react';
-import { head, filter, get } from 'lodash';
+import { find, endsWith, some, } from 'lodash';
+import url from 'url';
 
 /**
  * Internal Dependencies
  */
 import FeaturedVideo from './featured-video';
 import FeaturedImage from './featured-image';
 
+const candidateForFeature = ( media ) => {
```

Comment:
why 
`const candidateForFeature = ( media ) =>` instead of good ole (and shorter)
`function candidateForFeature( media )`?